README.md

```diff
@@ -54,7 +54,7 @@ for items in items_of(client):
   emit("did something")
   # do something else
   emit("did something else")
-  context().stop()
+  context().stop() # or could be context().fail()
```

Comment:
At the top of this file, could we standardize "GumShoe" to "Gumshoe"?  I'm pretty sure it's all one word with no capital S).